mzLib/MzLibUtil/MrsNoiseEstimation/Level.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace SpectralAveraging;
```

Comment:
Consider removing the "MRS" parts of these namespaces

Reply:
The thought here was that theses are methods and classes that will not be used frequently, but could be useful in many places. Adding them to MzLibUtil would allow them to be called form anywhere, but relegating them to their own namespace would prevent them from being accessible unless actually needed. Should this be changed?

Reply:
Hmm, this is just a naming comment regarding `MrsNoiseEstimation` and possibly changing it to `NoiseEstimation` or something. No strong preference, just a thought.